### PR TITLE
Use common linter settings

### PR DESCRIPTION
**Pull request by Hyperkid123**

### CODE REVIEW

**Review by ryelo — DISMISSED** [review]

**Comment by Hyperkid123** [comment]
:no_good: these conflicts

**Review by ryelo — COMMENTED** [review]
I guess this is an overall comment, but it seems like there is a rule that adds a `,` at the end of a list, even if there is nothing to follow. I'm not sure if that's intentional or not, but IMO it'd be better to only add it if there's an item to follow

**Comment by Hyperkid123** [comment]
That is a prettier config and it is added on multiline arrays/objects. It can be configured I think (not all prettier configurations can). O honestly don't mind either way since the IDE will fix fit for me :smile:.

**Review by karelhala — DISMISSED** [review]

**Review by karelhala — APPROVED** [review]

File: src/js/App/RootApp.js [hunk 1]
```diff
@@ -3,73 +3,58 @@ import PropTypes from 'prop-types';
...
+    <Fragment>
+      <div
+        className="pf-c-drawer__content"
+        data-ouia-page-type={activeApp || appId}
+        data-ouia-page-group={activeLocation}
+        data-ouia-safe="true"
+        {...(pageAction && { 'data-ouia-page-action': pageAction })}
+        {...(pageObjectId && { 'data-ouia-page-object-id': pageObjectId })}
+      >
+        <div>
```
Comment (jtomasek) [inline src/js/App/RootApp.js:21]: Addition of this div breaks the PF Page + PageSection layout which uses flexbox. This currently breaks the Assisted Installer in OCM. It needs to be replaced with React.Fragment

Comment (karelhala) [reply src/js/App/RootApp.js:21]: This PR uses new linter, you might be refering to this PR https://github.com/RedHatInsights/insights-chrome/pull/1008

**Pull request merged**